src/conda_package_handling/api.py

```diff
@@ -1,19 +1,30 @@
-import os as _os
-from glob import glob as _glob
 import functools as _functools
+import os as _os
 import tempfile as _tempfile
+from glob import glob as _glob
 
+import conda_package_streaming.transmute
 import tqdm as _tqdm
+import zstandard
+
+from .conda_fmt import ZSTD_COMPRESS_LEVEL, ZSTD_COMPRESS_THREADS
+from .conda_fmt import CondaFormat_v2 as _CondaFormat_v2
 
 # expose these two exceptions as part of the API.  Everything else should feed into these.
 from .exceptions import ConversionError, InvalidArchiveError  # NOQA
-from .tarball import CondaTarBZ2 as _CondaTarBZ2, libarchive_enabled  # NOQA
-from .conda_fmt import CondaFormat_v2 as _CondaFormat_v2
-from .utils import (TemporaryDirectory as _TemporaryDirectory, rm_rf as _rm_rf,
-                    get_executor as _get_executor)
+from .tarball import CondaTarBZ2 as _CondaTarBZ2
+from .utils import TemporaryDirectory as _TemporaryDirectory
+from .utils import get_executor as _get_executor
+from .utils import rm_rf as _rm_rf
 
-SUPPORTED_EXTENSIONS = {'.tar.bz2': _CondaTarBZ2,
-                        '.conda': _CondaFormat_v2}
+SUPPORTED_EXTENSIONS = {".tar.bz2": _CondaTarBZ2, ".conda": _CondaFormat_v2}
+
+# check this instead of version
+# extract() is threadsafe but create() may not be
+THREADSAFE_EXTRACT = True
+
+# old API meaning "can extract .conda" (which we do without libarchive)
+libarchive_enabled = True
```

Comment:
With a little more effort we could use `.tar.bz2` only when `zstandard` was unavailable